store/CSDK/src/CarbonRow.cpp

```diff
@@ -170,6 +170,9 @@ char *CarbonRow::getString(int ordinal) {
     args[0].l = carbonRow;
     args[1].i = ordinal;
     jobject data = jniEnv->CallStaticObjectMethodA(rowUtilClass, getStringId, args);
+    if (jniEnv->ExceptionCheck()) {
```

Comment:
Exception check should be done for all other method calls also like getBoolean, getFloat,etc.

ArrayIndexOutofBoundsException on wrong index & cast exceptions  on wrong method usage are possible

Reply:
1. done, please check again.
2.no need create new jvm, it can be used again after catch. In the test case, I use one jvm and try catch exception in the first test case:  tryCarbonRowException. testCarbonProperties is running after try catch.

```
  tryCatchException(env);
        tryCarbonRowException(env, smallFilePath);
        testCarbonProperties(env);
        testWriteData(env, "./data", 1, argv);
```

Reply:
ok